gradle.properties

```diff
@@ -25,6 +25,9 @@ group=org.apache.kafka
 #  - streams/quickstart/java/pom.xml
 version=3.7.0-SNAPSHOT
 scalaVersion=2.13.12
+# Adding swaggerVersion in gradle.properties to have a single version in place for swagger
+# New version of Swagger 2.2.14 requires minimum JDK 11.
+swaggerVersion=2.2.8
```

Comment:
Yes, i agree @ijuma , but as per the https://github.com/gradle/gradle/issues/1697#issuecomment-506910915 , plugin block is highly constrained, as

1. Only the version accepts interpolated strings (id must be a string literal)
2. Replacement expressions are limited to variable references: this means that we cannot call methods or use sophisticated build logic to retrieve dependency versions. But anything declared in a gradle.properties file can be inserted into the version string

So, i made changes accordingly.

Apparently we can have another approach possible where we can add specific resolutionStrategy using the pluginManagement block in settings.gradle  and specify to use the specific plugin having the version defined in dependencies.gradle, this would work as well.

Your thoughts @ijuma